src/App.js

```diff
@@ -1,21 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './Component/Calculator';
+import './index.css';
 
 function App() {
```

Comment:
- [ ] Your App component is function-based, whereas your components should be class-based. Kindly refactor your component to be class-based

Reply:
Thank you for your review, changes implemented